testbed/testbed/load_generator.go

```diff
@@ -220,6 +227,55 @@ func (lg *LoadGenerator) generateTrace() {
 	}
 }
 
+func (lg *LoadGenerator) generateTrace() {
+	traceSender := lg.sender.(TraceDataSender)
+
+	itemsPerBatch := int(lg.options.ItemsPerBatch)
+
+	traceData := data.NewTraceData()
+	traceData.ResourceSpans().Resize(1)
+	traceData.ResourceSpans().At(0).InstrumentationLibrarySpans().Resize(1)
+	spans := traceData.ResourceSpans().At(0).InstrumentationLibrarySpans().At(0).Spans()
```

Comment:
Can we use temp vars to avoid repetition and make this more readable?

Reply:
updated